image/docker/schema2/list.py

```diff
@@ -209,6 +209,13 @@ def manifest_dict(self):
     def bytes(self):
         return self._manifest_bytes
 
+    @property
+    def filesystem_layers(self):
+        """
+        Returns the file system layers of this manifest, from base to leaf.
+        """
+        return None
```

Comment:
Technically that's incorrect, this information is stored in the image config. Why do you need this method?

Reply:
it's only here to satisfy the `ManifestInterface` base property, so it isn't strictly needed.
I should update the comment though, since this won't return any layers for manifests of type list.